install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -26,13 +26,13 @@ istio-ingressgateway:
   autoscaleMax: 5
   # specify replicaCount when autoscaleEnabled: false
   # replicaCount: 1
-  resources: {}
-    # limits:
-    #  cpu: 100m
-    #  memory: 128Mi
-    #requests:
-    #  cpu: 1800m
-    #  memory: 256Mi
+  resources:
```

Comment:
do you also want to set the concurrency here? not sure if our default of 2 is universal